src/main/kotlin/no/nav/familie/ef/mottak/integration/SaksbehandlingClient.kt

```diff
@@ -0,0 +1,32 @@
+package no.nav.familie.ef.mottak.integration
+
+import no.nav.familie.http.client.AbstractPingableRestClient
+import no.nav.familie.kontrakter.ef.felles.StønadType
+import no.nav.familie.kontrakter.felles.PersonIdent
+import no.nav.familie.kontrakter.felles.Ressurs
+import org.springframework.beans.factory.annotation.Qualifier
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Service
+import org.springframework.web.client.RestOperations
+import org.springframework.web.util.UriComponentsBuilder
+import java.net.URI
+
+@Service
+class SaksbehandlingClient(@Value("\${EF_SAK_URL}")
+                           private val uri: URI,
+                           @Qualifier("restTemplateAzure")
+                           restOperations: RestOperations)
+    : AbstractPingableRestClient(restOperations, "saksbehandling") {
+
+    override val pingUri: URI = UriComponentsBuilder.fromUri(uri).pathSegment("api/ping").build().toUri()
+
+    fun finnesBehandlingForPerson(stønadType: StønadType, personIdent: String): Boolean {
```

Comment:
Navngivning. Høres litt ut som om denne skulle returnere trur dersom det finnes en behandling i infotrygd. Vet ikke om jeg har noen gode alternativer, men her er noen ugjennomtenkte ideer: 
finnesBehandlingForPersonIEfSak
finnesBehandlingForPersonINyLøsning 
behandlingMigrertUtAvInfotrygd
behandlingMigrertTilNyLøsning

Reply:
Det gir kanskje mest mening det neste året, men `finnesBehandlingForPersonINyLøsning` er veldig eksplisitt på hva som er meningen her. Så bør den isåfall renames den dagen infotrygd er ute av EF-verden...

Reply:
Da blir vel hele sjekken litt meningløs og kan fjernes?